cms/search/views.py

```diff
@@ -77,6 +77,25 @@ def search(_class):
         query_string, search_ordering, search_type, date_from, date_to
     )
 
+    # figure out the page results from - to numbers
+    if page > 0:
+        min_result_index = min(
+            1 + ((page - 1) * settings.SEARCH_RESULTS_PER_PAGE), search_results_count
+        )
+        max_result_index = min(
+            page * settings.SEARCH_RESULTS_PER_PAGE, search_results_count
+        )
+    else:
+        # 0 and negative numbers count down from the last result
+        abs_min_result_index = (
```

Comment:
If there's 5 pages, how does this handle page -7? 😛 
This is cool but I feel like it's unnecessary.